### PR TITLE
click to scroll prompt button

### LINKED ISSUE #152: Improve UX of scroll prompt
Make them
- scroll when clicked
- animated (if got time) 
- Not to overlap original screen contents

### PR DESCRIPTION
**Pull request by hzxcaryn**
WIP
closes #152

### CODE REVIEW

File: Web/src/components/common/ScrollDownButton.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+const ScrollDownButton: React.FC<ScrollDownButtonProps> = ({
+  promptText,
+  scrollToRef,
+  backgroundColor,
+  sticky = false,
+}) => {
+  const scrollRefIntoView = () => {
+    if (scrollToRef.current != null) {
+      smoothscroll.polyfill();
```
Comment (indocomsoft) [inline Web/src/components/common/ScrollDownButton.tsx]: Are we gonna move this to App.tsx?

File: Web/src/components/create-room/HostRoom.tsx [hunk 2]
```diff
@@ -143,10 +135,20 @@ const PindaHappy = styled(PindaHappySVG)`
...
+    return contentRef.current.clientHeight > (window.innerHeight - spaceAroundContent);
+  }
+  return false;
+};
+
 const HostRoomLobby: React.FC<FinishedComponentProps> = ({
   room, error, users, allMetas, resultMeta,
 }) => {
   const comm = useContext(CommContext);
+  const membersListRef = useRef<HTMLDivElement>(null);
```
Comment (indocomsoft) [inline Web/src/components/create-room/HostRoom.tsx]:
```suggestion
  const membersListRef = useRef<HTMLDivElement>();
```

Comment (indocomsoft) [reply Web/src/components/create-room/HostRoom.tsx]:
```suggestion
  const membersListRef = useRef<HTMLDivElement>();
```

File: Web/src/components/common/ScrollDownButton.tsx (hunk 1 above)
Comment (hzxcaryn) [reply Web/src/components/common/ScrollDownButton.tsx]: oh yahor

File: Web/src/components/create-room/HostRoom.tsx (hunk 2 above)
Comment (hzxcaryn) [reply Web/src/components/create-room/HostRoom.tsx]: Not changing cos 
```
Type 'MutableRefObject<HTMLDivElement | undefined>' is not assignable to type 'RefObject<HTMLDivElement>'
```

**Pull request merged**